compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -36,7 +36,9 @@ static llvm::cl::opt<AIEPassPipeline> clUsePipeline(
             "Use the IREE lowering to AIR dialect through pack operation"),
         clEnumValN(AIEPassPipeline::PadPackPipeline, "pad-pack",
                    "Use the IREE lowering to AIR dialect through "
-                   "pad and pack operations")),
+                   "pad and pack operations"),
+        clEnumValN(AIEPassPipeline::ObjectFifoPipeline, "objectFifo",
+                   "Use the IREE lowering to objectFifos")),
```

Comment:
This is kind of confusing to put objectFifo pipeline with pack-peel pipelines, since it's actually using pack-peel pipeline. Maybe add another flag just to switch between objectfifo and MLIR-AIR?